### PR TITLE
pythonPackages.alerta-server: refactor fix broken build

### PR DESCRIPTION
**Pull request by costrouc**
###### Motivation for this change

Enable all tests and fix broken builds in staging-next. In this case fix alerta-server and some of its dependencies

###### Things done

- [X] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [X] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [X] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

###### Notify maintainers

cc @

### CODE REVIEW

File: pkgs/development/python-modules/lupa/default.nix [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+    pkg-config
+  ];
+
+  buildInputs = [
+    luajit
+  ];
+
+  meta = with lib; {
+    description = "Python wrapper around Lua and LuaJIT";
+    homepage = https://github.com/scoder/lupa;
```
Comment (doronbehar) [inline pkgs/development/python-modules/lupa/default.nix:31]:
```suggestion
    homepage = "https://github.com/scoder/lupa";
```

https://github.com/NixOS/rfcs/pull/45

File: pkgs/development/python-modules/fakeredis/default.nix [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+
+  src = fetchFromGitHub {
+    owner = "jamesls";
+    repo = "fakeredis";
+    rev = version;
+    sha256 = "0j2y7mxdp63ida5f5l9hc75sjz1rqdz2dm4zairg0aksqrz197qa";
+  };
+
+  propagatedBuildInputs = [
+    redis
```
Comment (doronbehar) [inline pkgs/development/python-modules/fakeredis/default.nix:24]: Perhaps this should go to `buildInputs`?

**Review by doronbehar — CHANGES_REQUESTED** [review]

**Comment by doronbehar** [comment]
Note the merge conflicts.

**Comment by dotlambda** [comment]
See also https://github.com/NixOS/nixpkgs/pull/112125.

**Comment by doronbehar** [comment]
Closing due to op not responding and issue seems to be fixed.

**Pull request closed without merging**